--- origsrc/gtkglext-1.2.0/examples/Makefile.am	2003-09-09 04:50:04.000000000 -0500
+++ src/gtkglext-1.2.0/examples/Makefile.am	2008-12-09 22:00:10.034250000 -0600
@@ -18,10 +18,7 @@
 	-I$(top_builddir)/gtk		\
 	$(GTKGLEXT_DEBUG_FLAGS)		\
 	$(EXAMPLES_DEP_CFLAGS)		\
-	-DG_DISABLE_DEPRECATED		\
-	-DGDK_DISABLE_DEPRECATED	\
-	-DGDK_PIXBUF_DISABLE_DEPRECATED	\
-	-DGTK_DISABLE_DEPRECATED
+	$(NULL)
 
 DEPS = \
 	$(top_builddir)/gdk/$(gdkglext_targetlib)	\
@@ -67,7 +64,7 @@
 win32_programs = \
 	wglinfo
 
-if PLATFORM_WIN32
+if OS_WIN32
 EXTRA_PROGRAMS = $(common_programs) $(win32_programs)
 else
 EXTRA_PROGRAMS = $(common_programs) $(x11_programs)
